wsit/hudson/quicklook.sh

```diff
@@ -108,17 +108,38 @@ _unzip $GF_ZIP $GF_WORK_DIR
 
 if [ -d "$GF_WORK_DIR/glassfish4" ]; then
     SERVER_DIR=glassfish4
+elif [ -d "$GF_WORK_DIR/glassfish5" ]; then
+    SERVER_DIR=glassfish5
 else
     SERVER_DIR=glassfish3
 fi
 
+AS_HOME=$GF_WORK_DIR/$SERVER_DIR/glassfish
+
+echo "AS_HOME: $AS_HOME"
+
+#domain setup
+echo "Re-creating default domain"
+create_domain
```

Comment:
so this is going to create new domain, later on (LInes 133-142) it is being altered...
Sth like following would do the same job:

```
#this can actually be set externally but if required to be explicitly overridden within the script
INSTANCE_PORT=7998
set_ports
create_domain 
```

and you're done. Shorter, cleaner... important is ordering of these calls